### PR TITLE
8 add question modal

### LINKED ISSUE #8: Add: question modal
Is your feature request related to a problem? Please describe.
N/A

Describe how the solution should behave
The add question modal should be in the `components` directory and displayed with a dark overlay once called from any other component.

Screenshots of the design/if any
image

Additional context
Formik for validation??
N/A

### PR DESCRIPTION
**Pull request by suateneskoc**
screenshot

### CODE REVIEW

File: src/components/FormField/FormField.jsx [hunk 1]
```diff
@@ -3,19 +3,38 @@
...
+      ) : (
+        <Field
+          autoComplete={autoComplete}
+          className="my-1 focus:ring-red-500 focus:border-red-500 block w-full shadow-sm sm:text-sm border-gray-300 rounded-md"
+          id={name}
+          name={name}
+          type={type}
+        />
+      )}
+
```
Comment (MuhannedNoman) [inline src/components/FormField/FormField.jsx]: Let's make this an if/else condition statement, more readable.

File: src/components/modals/AddQuestion/AddQuestion.jsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (MuhannedNoman) [inline src/components/modals/AddQuestion/AddQuestion.jsx]: Where did you need to spread? Which line is causing this?

**Review by MuhannedNoman — CHANGES_REQUESTED** [review]
@suateneskoc nice work here, just left some comments to make things better let me know what you think.

File: src/components/modals/AddQuestion/AddQuestion.jsx (hunk 2 above)
Comment (suateneskoc) [reply src/components/modals/AddQuestion/AddQuestion.jsx]: React Dropzone library (file drag and drop) requires prop spreading

Comment (MuhannedNoman) [reply src/components/modals/AddQuestion/AddQuestion.jsx]: Since it is one line let's add an ignore for that line only.

**Review by MuhannedNoman — DISMISSED** [review]
LGTM @suateneskoc, resolve the spread thing and tag me again to merge.

**Review by MuhannedNoman — APPROVED** [review]

**Pull request merged**